intake/cli/server/__main__.py

```diff
@@ -39,6 +39,9 @@ def main(argv=None):
                         help='Name of catalog YAML file')
     parser.add_argument('--flatten', dest='flatten', action='store_true')
     parser.add_argument('--no-flatten', dest='flatten', action='store_false')
+    parser.add_argument('-a', '--address', type=str, 
+                        default=conf.get('address', 'localhost'),
+                        help='hosting address')
```

Comment:
Could be a bit more verbose here? Yes, I realise the other options are also very, er, concise.
Also, you might consider including this in the docs pages.

Reply:
It is in the docs pages, but I can make this more verbose.